jobs/generation/Utilities.groovy

```diff
@@ -189,15 +189,6 @@ class Utilities {
                                 // For outerloop runs.
                                 'outer-latest-or-auto':'ubuntu1610-20170216-outer',
                                 ],
-                            'OSX10.11' :
-                                [
-                                // Generic version label
-                                '' : 'mac',
```

Comment:
there were two element in the switch statement that had 10.11. I have no idea that was even possible.